analysis/Scripts/Universal_comparison.py

```diff
@@ -78,7 +79,8 @@
 
 histogram2.SetLineColor(9) # blue for tree2
 histogram2.SetLineWidth(1)
-extralabel1=" "+particle+"Data "+year+" " +MagPol #label for tree1 in the Legend
+#extralabel1=" "+particle+"Data "+year+" " +MagPol #label for tree1 in the Legend
+extralabel1=" XiC MC (26103091) 2017 MagUp)"
```

Comment:
The year seems to be hard-coded, but perhaps that's OK for current functionality.